sale_order_line_remove/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Remember to add copyrights on non-init files.

Reply:
done! :+1:  thank you